README.md

```diff
@@ -270,6 +271,17 @@ const renderedRichText = renderRichText(blok.text)
 <div set:html="{renderedRichText}"></div>
 ```
 
+Use the `<RichtextRenderer />` component if you are embedding `bloks` in your rich text:
+
+```jsx
+---
+import { RenderRichtext } from '@storyblok/astro';
+const { blok } = Astro.props
+---
+
+<RichtextRenderer richtextObject="{blok.richtext}" richtextOptions={options} />
+```
+
 You can also set a **custom Schema and component resolver** by passing the options as the second parameter of the `renderRichText` function:
```

Comment:
@Baheya I think here it would be helpful to point out that this also works when using the component, otherwise it may not be completely clear at a first glance what the `richtextOptions` prop of the component is for. Maybe you could even leave the prop out in the first example showing the `RichtextRenderer` component. Then you could show it as a specific example of using a custom schema/resolver with the component. What do you think?